extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -289,11 +286,11 @@ private function getContributorsDetails( $contributors ) {
 				'userName' => $userName,
 				'avatar' => $avatar,
 				'contributionsText' => $this->msg( 'communitypage-contributions' )
-					->numParams( $this->getLanguage()->formatNum( $contributor[ 'contributions' ] ) )->text(),
+					->numParams( $this->getLanguage()->formatNum( $contributor[ 'contributions' ] ?? 0 ) )->text(),
 				'profilePage' => $user->getUserPage()->getLocalURL(),
 				'count' => $count,
-				'isAdmin' => $contributor['isAdmin'],
-				'timeAgo' => $contributor['timeAgo'],
+				'isAdmin' => $contributor['isAdmin'] ?? false,
+				'timeAgo' => $contributor['timeAgo'] ?? null,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "timeAgo" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)